### PR TITLE
Issue #SB-16203 fix: District name starting with small letter

### PR DESCRIPTION
**Pull request by JagadishPujari**

### CODE REVIEW

File: src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.ts [hunk 1]
```diff
@@ -230,6 +230,9 @@ export class DataChartComponent implements OnInit, OnDestroy {
     if (_.get(this.chartConfig, 'labels')) {
       labels = _.get(this.chartConfig, 'labels');
     }
+    _.forEach(labels, (label, key) => {
+      labels[key] = _.capitalize(label);
```
Comment (yravinderkumar33) [inline src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.ts:234]: This looks good, but you need to do the same when showing in filters also.
There also labels should in be title case.

Comment (JagadishPujari) [reply src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.ts:234]: That is already fixed and PR is merged

Comment (yravinderkumar33) [reply src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.ts:234]: oh okay. Then it's perfect.

File: src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.spec.ts [hunk 2]
```diff
@@ -145,7 +145,7 @@ describe('DataChartComponent', () => {
             'Number of QR codes with no linked content': '60'
         }]);
         expect(component.noResultsFound).toBe(false);
-        expect(component.chartLabels).toEqual(['class 2']);
+        expect(component.chartLabels).toEqual(['Class 2']);
```
Comment (vinukumar-vs) [inline src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.spec.ts:148]: This is not a valid test-case.

**Pull request merged**